ChangeLog

```diff
@@ -59,6 +59,8 @@ ver. 0.9.4 (2015/XX/XXX) - wanna-be-released
      rest api and web interface (gh-1223)
    * Add *_backend options for services to allow distros to set the default
      backend per service, set default to systemd for Fedora as appropriate
+   * small improvement for better handling of many log files (gh-1265)
+     Thanks @kshetragia
```

Comment:
gr... I guess github didn't post my comment from the phone :-(
would be better IMHO to provide a bit more specific than "small improvement" description, e.g.

"Use associative array (dict) for monitored log files to speed up lookup operations. Provides performance improvements while monitoring large number of files ..."